WordPress/WordPressShareExtension/ShareModularViewController.swift

```diff
@@ -244,6 +246,22 @@ extension ShareModularViewController {
         reloadSitesIfNeeded()
     }
 
+    func showPostTypePicker() {
+        guard isPublishingPost == false else {
+            return
+        }
+
+        let typePicker = SharePostTypePickerViewController(postType: shareData.postType)
+        typePicker.onValueChanged = { [weak self] postType in
+            self?.tracks.trackExtensionPostTypeSelected(postType.rawValue)
+            self?.shareData.postType = postType
+            self?.refreshModulesTable()
```

Comment:
As none of these lines will run when `self` is `nil` then maybe we can unwrap self on top and bail out immediately.